src/DetailsView/details-view-initializer.ts

```diff
@@ -195,7 +195,13 @@ if (isNaN(tabId) === false) {
                 new NavigatorUtils(window.navigator).getBrowserSpec(),
                 assessmentDefaultMessageGenerator,
             );
-            const reportGenerator = new ReportGenerator(reportNameGenerator, reportHtmlGenerator, assessmentReportHtmlGenerator);
+
+            const reportGeneratorProvider = createReportGeneratorProvider(
+                reportNameGenerator,
+                reportHtmlGenerator,
```

Comment:
Do you want to rename this to reflect this is the report html generator for automated checks?

Reply:
It includes both actually... I think this is a User Story we can thing about it at the end of the feature (have some sort of separation from the two reports we have now)